client/components/category/SingleCategory.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { selectCategory } from '../../redux/singleCategory'
+
+class SingleCategory extends Component {
+  componentDidMount() {
+    //this.props.selectCategory(this.props.selectedId)
+  }
+  render() {
+    const { selectedCategory } = this.props;
+    // should be fleshed out more based on api routes to include all costumes belonging to a category
```

Comment:
Label comments like these `// TODO: {task_here}` - editors support this and will allow you to quickly find them.